### PR TITLE
implemented edit and archive

### PR DESCRIPTION
**Pull request by LeenMass**

### CODE REVIEW

**Review by Karyum — DISMISSED** [review]

**Comment by tawfiknasser** [comment]
The only one thing I'm worrying about is dropping others changes. Wouldn't it be better to set a default value to the site mix?
This issue is blocking you for two days. Instead of solving it. We are moving around it and avoiding to fix it. Please schedule a meeting between you. @LeenMass and @skittan4 . Have a conversation and end with action points.

**Comment by tawfiknasser** [comment]
Setting the default value to an empty object `func(obj={})` and updating the database should solve all your issues. 5 minutes fix

**Comment by LeenMass** [comment]
although the planned mix is not supposed to be in that table (the projects table) I will do the change but first.
We already have the sites table I don't know why we need this column, and you didn't explain to me yet whats the different between the sites table and the sites column, why not have another table connecting 2 of the sites?
and again are you sure you want the column for the sites ?

File: router.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const projectController = require('./controllers/projects.controller')
 const loginController = require('./controllers/login.controller')
 const authorizeMiddleware = require('./middleware/authorization')
 const verifyToken = require('./middleware/verifyUser')
-
+const auth = require('./controllers/auth.controller')
```
Comment (tawfiknasser) [inline router.js:7]: not used var?

**Review by tawfiknasser — APPROVED** [review]

**Pull request merged**